books/management/commands/addepub.py

```diff
@@ -249,7 +249,7 @@ def process_epub(self, filename, use_symlink=False):
                                 .split(',')
                             for tag in subject_split:
                                 if tag is not ' ':
-                                    tag = tag.encode("utf-8").lower().strip()
+                                    tag = tag.lower().strip()
```

Comment:
Hmmm, the problematic word on that quote is probably "**may**", which does not enforce that guideline as strongly as we'd like. I'll go with the nested try..except solution for the moment, in order to at least not have the import crash due to a tag not being able to be created.